pkg/controller.v1beta1/trial/managerclient/managerclient.go

```diff
@@ -35,6 +35,31 @@ func (d *DefaultClient) GetTrialObservationLog(
 	if err != nil {
 		return nil, err
 	}
+	if reply.ObservationLog == nil {
+		reply.ObservationLog = &api_pb.ObservationLog{}
+	}
+	if reply.ObservationLog.MetricLogs == nil {
+		reply.ObservationLog.MetricLogs = []*api_pb.MetricLog{}
+	}
+	// fetch additional metrics if exists
+	metricLogs := reply.ObservationLog.MetricLogs
+	for _, metricName := range instance.Spec.Objective.AdditionalMetricNames {
```

Comment:
Is that correct to add AdditionalMetricNames in `GetTrialObservationLog` function?
I believe this function is needed to add  `Observation` to Trial instance, right?
Currently, Trial `Observation` is related only to `ObjectiveMetricName`.

Reply:
I think so.

Reply:
@andreyvelich Yes, I am trying to append addtional metrics into `Trial.Status.Observation`.  So, we can get full metrics of each trial via kubernetes API. Are you considering this modification will confuse our users/developers ?

Reply:
Ok, I am just worried if the Suggestions will work correct.
I checked here: https://github.com/kubeflow/katib/blob/master/pkg/suggestion/v1beta1/internal/trial.py#L33, we add only objective metric to target metric. So, I think it should work fine.